packages/ui-predicate-vue/src/ui-predicate-options.vue

```diff
@@ -1,17 +1,23 @@
 <template>
-  <div class="ui-predicate__col ui-predicate__col--option">
-    <component
-      :is="getUIComponent(UITypes.PREDICATE_REMOVE)"
-      @click.native="remove(predicate)"
-      :predicate="predicate"
-      :disabled="predicate.$canBeRemoved === false"
-    />
-    <component
-      :is="getUIComponent(UITypes.PREDICATE_ADD)"
-      @click.native="add(predicate)"
-      :predicate="predicate"
-      :is-in-add-compound-mode="isInAddCompoundMode"
-    />
+  <div class="ui-predicate__col">
+    <div class="ui-predicate__options">
```

Comment:
no need, see https://github.com/FGRibreau/ui-predicate/commit/4b0aff896fbf7ebc4363055bd62d1e9f0d27c166

Reply:
On build, in ui-predicate-options.vue, if there is 2 sibling dynamic component, the last one is not rendered. So i need to wrap each other in a div. And so, i prefer to use a plural class name on parent and singular on children (wrapper).